Backpack/src/main/java/net/skyscanner/backpack/calendar/view/MonthView.kt

```diff
@@ -486,21 +522,28 @@ internal class MonthView @JvmOverloads constructor(
   }
 
   private fun findDayOffset() =
-    (if (dayOfWeekStart < weekStart) dayOfWeekStart + numberOfDays else dayOfWeekStart) - weekStart
+    if (getNonDrawnDaysOffset() >= numberOfDaysInAWeek) {
+      0
+    } else {
+      var dayOffset = dayOfWeekStart - weekStart
+      if (dayOfWeekStart < weekStart)
```

Comment:
minor, but maybe we should not omit the curly braces

Reply:
Done